mage_integrations/mage_integrations/sources/base.py

```diff
@@ -178,6 +191,25 @@ def process(self) -> None:
         try:
             if self.should_test_connection:
                 self.test_connection()
+            elif self.load_sample_data:
+                catalog = self.catalog or self.discover(streams=self.selected_streams)
+                for stream in catalog.get_selected_streams(self.state):
+                    gen = self.load_data(stream, sample_data=True)
+                    if gen is not None \
+                        and self.pipeline_uuid is not None \
```

Comment:
ideally, we shouldn't use pipeline_uuid and block_uuid here. mage_integrations module just output the data. then mage_ai can parse the output and store the dataframe.